Documentation/project-docs/glossary.md

```diff
@@ -139,3 +139,7 @@ The .NET platform currently doesn't contain a cross-platform XAML-based UI stack
 * [WPFLight](https://github.com/ronnycsharp/WPFLight): An OSS project to create WPF on top of XNA/MonoGame.
 * [Perspex](https://github.com/Perspex/Perspex): A cross-platform UI framework based on WPF.
 * [Granular](https://github.com/yuvaltz/Granular): A OSS project to allow WPF applications to run in the browser.
+
+## Engineering system
+
+* **Helix**. It's a massive parallel general purpose job processing and result aggregation system running in cloud. The workitems that corefx sends to Helix are [xunit](https://github.com/xunit) tests. Tests result are shown through *Mission Control* reporting site https://mc.dot.net/
```

Comment:
@jkotas feel free to suggest better explanation, I'll copy/paste with no problem.

Reply:
Might be worth mentioning that to go to the test results in a PR from Azure Devops, you can click on the Send to Helix step in the build and the logs will have the URL.

Reply:
I tried(https://dev.azure.com/dnceng/public/_build/results?buildId=81155&view=logs) ...and links works but test list are empty `No rows currently visible. Please check your filters.` is it expected?

Reply:
You need to uncheck the "Show failures only" checkbox.

![image](https://user-images.githubusercontent.com/22899328/51867010-533ca900-22ff-11e9-9017-1f833be24ee9.png)

Reply:
Actually having to uncheck that box, is a good thing -> "There where no test failures on that test run" 😆